_sass/module/_money.scss

```diff
@@ -0,0 +1,12 @@
+.money {
+  color: $color-grey-3;
+  font-family: $font-family-serif;
+  text-align: right;
+}
+
+.money--sans {
+  //font-size: 1.6rem;
```

Comment:
Almost perfect SCSS, except you left this one line commented out :P

Reply:
GAME OVER